usr/src/uts/common/fs/zfs/vdev.c

```diff
@@ -159,7 +159,7 @@ vdev_dbgmsg_print_tree(vdev_t *vd, int indent)
 	}
 
 	zfs_dbgmsg("%*svdev %u: %s%s, guid: %llu, path: %s, %s", indent,
-	    "", vd->vdev_id, vd->vdev_ops->vdev_op_type,
+	    "", (int)vd->vdev_id, vd->vdev_ops->vdev_op_type,
```

Comment:
zdb uses ```%llu``` with ```u_longlong_t```
Should we do that for consistency, which is also less risky given the definition of vdev_id?

Reply:
I think that either can work, but just adding the int cast seemed like a simpler change.
And I cannot imagine a pool with billions of vdevs.